www/docs/client/links/httpSubscriptionLink.md

```diff
@@ -174,6 +174,16 @@ const trpc = createTRPCClient<AppRouter>({
 });
 ```
 
+## Compatibility (React Native) {#compatibility-React Native}
+
+The `httpSubscriptionLink` makes use of the `EventSource` API, Streams API, and `AsyncIterator`s, these are not natively supported by React Native and will have to be polyfilled.
+
+To polyfill `EventSource` we recommend to use a polyfill that utilizes the networking library exposed by React Native, over using a polyfill that using the `XMLHttpRequest` API. Libraries that polyfill `EventSource` using `XMLHttpRequest` fail to reconnect after the app has been in the background. Consider using the [rn-eventsource-reborn](https://www.npmjs.com/package/rn-eventsource-reborn) package.
+
+The Streams API can be polyfilled using the [web-streams-polyfill](https://www.npmjs.com/package/web-streams-polyfill) package.
+
+`AsyncIterator`s can be polyfilled using the [@azure/core-asynciterator-polyfill](https://www.npmjs.com/package/@azure/core-asynciterator-polyfill) package.
+
```

Comment:
Would be great to add an installation snippet of the npm modules and example code where you put the polyfill ☺